src/components/TodosLogic.js

```diff
@@ -0,0 +1,30 @@
+import InputTodo from './InputTodo';
+import TodosList from './TodosList';
+
+const TodosLogic = () => {
+  const todos = [
+    {
+      id: 1,
+      title: 'Setup development environment',
+      completed: true,
+    },
+    {
+      id: 2,
+      title: 'Develop website and add content',
+      completed: false,
+    },
+    {
+      id: 3,
+      title: 'Deploy to live server',
+      completed: false,
+    },
+  ];
+  return (
+    <div>
+      <InputTodo />
+      <TodosList todosProps={todos} />
+    </div>
```

Comment:
### Requested changes

- [ ] In your TodosLogic, you have hardcoded some of the list items, you can remove those now as your project is ready for presentation.
- [ ] In the Header, you have stated that list items will persist in the local storage, but you have not added any code to save them in local storage. I'd suggest you can either remove that statement or add a function to save the data in local storage